en/sensor/airspeed.md

```diff
@@ -39,7 +38,21 @@ Enable each type using its [corresponding parameter](../advanced_config/paramete
 - **TE MS5525:** [SENS_EN_MS5525DS](../advanced_config/parameter_reference.md#SENS_EN_MS5525DS)
 - **Eagle Tree airspeed sensor:** [SENS_EN_ETSASPD](../advanced_config/parameter_reference.md#SENS_EN_ETSASPD)
 
-### Sensor-specific configuration
+You should also check [ASPD_PRIMARY](../advanced_config/parameter_reference.md#ASPD_PRIMARY) is `1` (see next section - this is the default).
+
+### Multiple Airspeed Sensors
+
+If you have multiple airspeed sensors then you can select which sensor is _preferred_ as the primary source using [ASPD_PRIMARY](../advanced_config/parameter_reference.md#ASPD_PRIMARY), where `1`, `2` and `3` reflect the order in which the airspeed sensors were started:
+- `-1`: Disabled (no airspeed information used). 
+- `0`: Synthetic airspeed estimation (groundspeed minus windspeed)
```

Comment:
The [EKF2 docs](http://docs.px4.io/main/en/advanced_config/tuning_the_ecl_ekf.html#synthetic-sideslip) on synthetic sideslip say "is enabled by setting the [EKF2_FUSE_BETA](http://docs.px4.io/main/en/advanced_config/parameter_reference.html#EKF2_FUSE_BETA) parameter to `1`"

So what's happening here? Do we also need to set ASPD_PRIMARY?

Reply:
No it's two different things. ASPD_PRIMARY (one could challenge the name...) set to 0 will make the selector output a "approximated" airspeed value, while setting the FUSE_BETA to 1 will just also EKF to estimate the wind without an airspeed sensor present. 
@bresch @RomanBapst would you know of a case where FUSE_BETA actually shouldn't be turned on a FW or VTOL in FW flight? Is it even necessary to keep this parameter? I haven't set up any vehicle where it wasn't enabled.

Reply:
OK. I will ignore this case.

Reply:
I would say that almost all planes have enough positive yaw stiffness (i.e.: directional stability) to estimate wind using sideslip fusion. The only case where it should be disabled is for a fighter aircraft with an unstable yaw axis or a skid-to-turn drone where the zero sideslip assumption is too far from the truth during turns.